### PR TITLE
Filter by serious and a high risk level only

### PR DESCRIPTION
**Pull request by StupidCodeFactory**
## CHANGES:

Added a checkbox to filter case by risk level

### CODE REVIEW

**Comment by StupidCodeFactory** [comment]
[(3) Filter cases by risk level](https://trello.com/c/SanjgRrB/367-3-filter-cases-by-risk-level)

File: spec/features/filter_investigations_spec.rb [hunk 1]
```diff
@@ -122,6 +125,21 @@
     expect(page).not_to have_listed_case(other_team_investigation.pretty_id)
   end
 
+  scenario "Filtering by risk-level cases only" do
+    check "Serious and high risk cases only"
+    click_on "Apply filters"
+    expect(page.find_field("Serious and high risk cases only")).to be_checked
```
Comment (frankieroberto) [inline spec/features/filter_investigations_spec.rb:131]:
```suggestion
    expect(page).to have_checked_field("Serious and high risk cases only")
```

File: app/views/investigations/_serious_and_high_risk_level_checkbox.html.erb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+           key: :risk_level,
+           classes: "govuk-checkboxes--small",
+           items: [
+             {
+               key: "serious_and_high_risk_level_only",
+               value: "yes",
+               unchecked_value: "no",
+               text: t(:serious_and_high_risk_level_only, scope: "cases.filters")
+             }
+           ] %>
```
Comment (frankieroberto) [inline app/views/investigations/_serious_and_high_risk_level_checkbox.html.erb:12]: ✂️

File: app/views/investigations/_coronavirus_and_risk_level_checkbox.html.erb [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<%= render "form_components/govuk_checkboxes",
+           form: form,
+           key: :coronavirus_related,
```
Comment (frankieroberto) [inline app/views/investigations/_coronavirus_and_risk_level_checkbox.html.erb:3]: Is this needed? ✂️ ?

**Pull request merged**